pkgs/development/haskell-modules/patches/hunspell-1.6.patch

```diff
@@ -0,0 +1,30 @@
+diff -Naur hunspell-hs-0.1.0.0.orig/hunspell-hs.cabal hunspell-hs-0.1.0.0/hunspell-hs.cabal
```

Comment:
It might be nicer to obtain the patch with `fetchpatch` from the commit in the PR.

Reply:
That would include the `extra-source-files:` line mentioning files, which aren't present on hackage. I think Cabal would stumble about that, but I can try it.